src/core/backbone_router/local.cpp

```diff
@@ -145,6 +145,7 @@ otError Local::SetConfig(const BackboneRouterConfig &aConfig)
     // Validate configuration according to Thread 1.2.1 Specification 5.21.3.3:
     // "The Reregistration Delay in seconds MUST be lower than (0.5 * MLR Timeout). It MUST be at least 1."
     VerifyOrExit(aConfig.mReregistrationDelay >= 1, error = OT_ERROR_INVALID_ARGS);
+    VerifyOrExit(aConfig.mMlrTimeout >= Mle::kMlrTimeoutMin, error = OT_ERROR_INVALID_ARGS);
```

Comment:
nit: how about adjusting the order a bit to put `mReregistrationDelay` relative checks together and match the comment?

Reply:
The current intention is to check two individual values first, and then check the relationship between them. Does it make sense to you?

Reply:
revised according